charts/prometheus-blackbox-exporter/values.yaml

```diff
@@ -0,0 +1,114 @@
+restartPolicy: Always
+
+kind: Deployment
+
+podDisruptionBudget: {}
+  # maxUnavailable: 0
+
+strategy:
+  rollingUpdate:
+    maxSurge: 1
+    maxUnavailable: 1
+  type: RollingUpdate
+
+image:
+  repository: prom/blackbox-exporter
+  tag: v0.16.0
```

Comment:
@jpweber we will need an image here so we can apply any CVE patches.

Reply:
Got it, I'll get that done soon.

Reply:
Work done on PR https://github.com/astronomer/ap-vendor/pull/12

Reply:
blackbox-exporter image work has been merged in to the ap-vendor repo. This PR has been updated to include the new astronomer image rather than the standard public one.